P/protozero/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "protozero"
+version = v"1.7.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/mapbox/protozero.git", "f379578a3f7c8162aac0ac31c2696de09a5b5f93")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/protozero
+
+mkdir build && cd build
+
+cmake .. \
+    -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release
+
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [AnyPlatform()]
+
+# The products that we will ensure are always built
+# protozero is a header only library, so no binary products
+products = Product[]
```

Comment:
No products?

Reply:
I was following the template for other header-only projects, like https://github.com/JuliaPackaging/Yggdrasil/blob/c063b0692d36ef5c41589a92738a50bfc94ef9ff/T/Taskflow/build_tarballs.jl#L33. Is it better to include file products, and if so, do I need to have a file product for every `*.hpp` generated?